### ISSUE TITLE
[Azure] Resource group and vnet name not parameterized

### ISSUE THREAD

**Issue by wrast** [pivotal-cf/pcf-pipelines#184]
Resource group is hardcoded to `network-core` and vnet-name is hardcoded to `vnet-pcf`. These should be parameterized.

https://github.com/pivotal-cf/pcf-pipelines/blob/6ba84fb4e83059bedffcb42c1ea48b56b31f5b52/install-pcf/azure/tasks/create-infrastructure/task.sh#L12

**Comment by abbyachau** [comment]
hey thanks @wrast. have you folks implemented a workaround for this issue?

**Comment by wrast** [comment]
Hey @abbyachau. This is the change we made to that line:

```
ERT_SUBNET_CMD="az network vnet subnet list -g ${AZURE_TERRAFORM_PREFIX} --vnet-name ${AZURE_TERRAFORM_PREFIX}-virtual-network --output json | jq '.[] | select(.name == \"ert\") | .id' | tr -d '\"'"
```

Looking at this again, this command apparently only matters when executed while using the multi-resgroup pipeline. The comment on line 9 suggests it's only needed for multi-resgroup. We felt like this line was still wrong though so we made the change anyway.

The end result with and without the change is that `ERT_SUBNET` evaluates to empty, which in our case is correct. But it may not be correct when using the multi-resgroup pipeline.
